src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -13179,11 +12815,10 @@ private PostSkipAction SkipBadOrderingListTokens(SeparatedSyntaxListBuilder<Orde
             CSharpSyntaxNode tmp = null;
             Debug.Assert(list.Count > 0);
             return this.SkipBadSeparatedListTokensWithExpectedKind(ref tmp, list,
-                p => p.CurrentToken.Kind != SyntaxKind.CommaToken,
-                p => p.CurrentToken.Kind == SyntaxKind.CloseParenToken
+                static p => p.CurrentToken.Kind != SyntaxKind.CommaToken,
```

Comment:
It looks like it would be fine to make this SkipBadOrderingListTokens a local function. No need to do in this PR.

Reply:
Moved :)